src/main/java/frc/robot/commands/PressIndex.java

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.subsystems.Index;
+
+/** An example command that uses an example subsystem. */
+public class PressIndex extends Command {
+  @SuppressWarnings({"PMD.UnusedPrivateField", "PMD.SingularField"})
+  private final Index index;
+
+  /**
+   * Creates a new PressIndex.
+   *
+   * @param subsystem The subsystem used by this command.
+   */
+  public PressIndex(Index index) {
+    this.index = index;
+    addRequirements(index);
+    // Use addRequirements() here to declare subsystem dependencies.
+  }
+
+  // Called when the command is initially scheduled.
+  @Override
+  public void initialize() {
+    index.press();
+  }
+
+  // Called every time the scheduler runs while the command is scheduled.
+  @Override
+  public void execute() {}
+
+  // Called once the command ends or is interrupted.
+  @Override
+  public void end(boolean interrupted) {}
+
+  // Returns true when the command should end.
+  @Override
+  public boolean isFinished() {
+    return false;
```

Comment:
this needs a isFinished, or it will keep running.
the isFinished should probably be based on the sensor being hit.